qcodes/instrument/parameter.py

```diff
@@ -1745,6 +1695,97 @@ def setpoint_full_names(self) -> Optional[Sequence[Sequence[str]]]:
             return self.setpoint_names
 
 
+class _Cache:
+    def __init__(self,
+                 parameter: '_BaseParameter',
+                 max_val_age: Optional[float] = None):
+        self._parameter = parameter
+        self._value: ParamDataType = None
+        self._raw_value: ParamRawDataType = None
+        self._timestamp: Optional[datetime] = None
+        self._max_val_age = max_val_age
+
+    @property
+    def timestamp(self) -> Optional[datetime]:
+        return self._timestamp
+
+    @property
+    def max_val_age(self) -> Optional[float]:
+        return self._max_val_age
+
+    def set(self, value: ParamDataType) -> None:
+        """
+        Set the cached value of the parameter without invoking the
+        ``set_cmd`` of the parameter (if it has one). For example, in case of
+        an instrument parameter, calling :meth:`cache.set` as opposed to
+        calling ``set`` will only change the internally-stored value of
+        the parameter (that is available when calling ``cache.get`` or
+        ``get_latest``), and will pass that value to the instrument.
```

Comment:
and will not pass that value to the instrument?

Reply:
indeed. it's a typo.